ddtrace/contrib/botocore/services/bedrock.py

```diff
@@ -325,12 +334,11 @@ def patched_bedrock_api_call(original_func, instance, args, kwargs, function_var
     pin = function_vars.get("pin")
     endpoint_name = function_vars.get("endpoint_name")
     integration = function_vars.get("integration")
-    # This span will be finished separately as the user fully consumes the stream body, or on error.
-    bedrock_span = pin.tracer.start_span(
+    # This span will be finished once the user fully consumes the stream body, or on error.
```

Comment:
What changed so that we want to use `trace` instead of `start_span` with `activation=False`?

Reply:
This is a subtle but pretty important change and has to do with trace parenting. Basically revolves around the idea of what span is "active" (which spans will automatically then be children from).

This change switches to using `tracer.trace()` instead of `tracer.start_span(..., activate=False)` for bedrock spans, because the latter method meant that bedrock spans would always be root spans (messing up parenting for traces containing non-root bedrock spans). Now bedrock spans can correctly be created as children of whatever span is active.
Also, by getting rid of the activate=False argument, this means that bedrock spans will now continue to be the active span until the stream/body is completely consumed. Previously we allowed bedrock spans to not be active, meaning other downstream spans would be children of the bedrock span's parent span (or in a new trace).